RoboFile.php

```diff
@@ -553,9 +553,6 @@ public function publish($opts = ['force' => false]) {
   function changelogUpdate($opts = ['version-name' => null]) {
     $this->say("Updating changelog");
     $outputs = $this->getChangelogController()->update($opts['version-name']);
-    if($opts['quiet']) {
```

Comment:
I've removed this because you don't need to explicitly suppress output if you use `say`, `yell`, etc - it's done in the task runner itself.